tck/src/test/java/org/eclipse/microprofile/jwt/bridge/tck/container/jaxrs/RolesAllowedTest.java

```diff
@@ -90,6 +91,22 @@ public static void generateToken() throws Exception {
         token = TokenUtils.generateTokenString("/Token1.json");
     }
 
+    @RunAsClient
```

Comment:
The endpoint setup using the bridge annotation is missing. I suppose the related properties from the original MP JWT test should be converted to the annotation and  added to the test endpoint ?

Reply:
@sberyozkin Yes, of course. My apologies. As you had said on our last call, this may take a bit of time while I fix the TCK and try to get something written and running in Open Liberty to verify the behavior.

Reply:
@ayoho Np at all, sure, let me merge this one and open another issue to convert this test and optional ones to use the new annotation.